Cargo.toml

```diff
@@ -49,7 +49,7 @@ cortex-m-rt = { version = "0.6.12", default-features = false }
 cortex-m-semihosting = { version = "0.5.0", default-features = false }
 crc = { version = "3.0.0", default-features = false }
 ctrlc = { version = "3.1.5", default-features = false }
-derive_more = { version = "0.99", default-features = false, features = ["from"] }
+derive_more = { version = "0.99", default-features = false, features = ["from", "display"] }
```

Comment:
Is `display` needed?

Reply:
It's not and I thought I deleted it, let me check what's up.